Dockerfile

```diff
@@ -1,17 +1,32 @@
 FROM mcr.microsoft.com/cbl-mariner/base/core:2.0
```

Comment:
Is the 3.0 ready to use?

Reply:
not yet, I have confirmed with their team. Also try to search here https://mcr.microsoft.com/en-us/catalog?search=mariner&type=partial&page=1.